### PR TITLE
Examples: Allow deployment of es example in a different namespace

### PR DESCRIPTION
**Pull request by huikang**
- the openshif-logging space may be unavailable

### CODE REVIEW

File: docs/HACKING.md [hunk 1]
```diff
@@ -91,7 +91,9 @@ Deploy the resources for the operator, build the operator image, push the image
 Deploy the pre-requirements for the operator to function (i.e. CRD, RBAC, sample secret)
 
 ### deploy-example
-Deploy an example custom resource for a single node Elasticsearch cluster
+Install the operator and deploy an example custom resource for a single node
+Elasticsearch cluster in the defautl namespace `openshift-logging`. To deploy
```
Comment (ewolinetz) [inline docs/HACKING.md:95]: s/defautl/default

File: bundle/manifests/elasticsearch-operator.v4.6.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -50,7 +50,7 @@ metadata:
                       }
                     }
                   },
-                  "redundancyPolicy": "SingleRedundancy",
+                  "redundancyPolicy": "ZeroRedundancy",
```
Comment (ewolinetz) [inline bundle/manifests/elasticsearch-operator.v4.6.0.clusterserviceversion.yaml:53]: :tada: thank you for finally fixing this example!

**Comment by ewolinetz** [comment]
one quick nit otherwise :shipit: 
/approve

File: docs/HACKING.md (hunk 1 above)
Comment (huikang) [reply docs/HACKING.md]: Fixed. Thanks.

**Pull request merged**